requirements/dev-requirements.txt

```diff
@@ -10,6 +10,8 @@ alembic==1.4.3
     # via -r base-requirements.in
 amqp==2.6.1
     # via kombu
+appnope==0.1.2
+    # via ipython
```

Comment:
At some point `make requirements` removed these two lines for you automatically, but it seems like that's no longer the case. I've raised that with the dependency pod, but in the meantime, could you remove this pair of lines and the ones like it in the other files (the ones about `appnope`)? That should fix the build.

Reply:
All the builds are failing because `RECAPTCHA_PRIVATE_KEY` and `RECAPTCHA_PUBLIC_KEY` in settings.py are `None` instead of a string, but this is necessary for it to work properly in production